### ISSUE TITLE
Automated Living Wood/ Living Rock Botania

### ISSUE THREAD

**Issue by knoxz** [AppliedEnergistics/Applied-Energistics-2#1607]
Hi there,

Agrarian Skies 2 V. 1.0.6
I created an automated way to create living wood and rock with just applied energistics. It works as follows.
I have 2 subsystems.

One with an import bus on a chest and formation planes and the 2. with annihilation planes and storage bus on barrels.
It imports the stone/wood from the chest, places it and when it is converted it will get sucked up from the annihilation planes. Only Living Wood and Rock can be imported by the annihilation planes. -> Barrels with Living rock and wood.
For stone this setup works really good. But for wood, it does not really work at all.
As I troubleshot it, I used a chest with a configured storage bus before I used barrels but its the same result. 

I have a problem with the system and wood of any kind. Sometimes it just vanishs. Sometimes it seems to get overriden. Sometimes it reappears and will become the wanted living wood...
I see wood vanish, I go afk for like 5 mins. I manually place wood and it becomes like instant living wood and seems to be harvested but actually stays and does not appear in the barrel, some times it does. There are happening loads of weird stuff with wood, but for stone everything works as intended.
If I put a stack of wood in the input chest, I'll get like 7 living wood sometimes even less.

![botania](https://cloud.githubusercontent.com/assets/7674778/8172184/66041892-13be-11e5-838e-3db594c3e42b.png)

**Comment by knoxz** [comment]
Ok might be a Cauldron issue. Just tested it in singleplayer and it works as intended even with wood.

**Comment by knoxz** [comment]
I tested the issue on a local forge server to see if the issue persists there. It was working. There were some plobs and it seemed as there are blocks getting replaced. But no blocks were vanishing and no blocks appeared in the output.
And then after a while it just stoped working at all. I did not change anything.
Even a server restart did not help.

![botaniasingleplayer2](https://cloud.githubusercontent.com/assets/7674778/8173206/5c288bbc-13ca-11e5-8b0e-a3c43a08bb6f.png)
